### ISSUE TITLE
Remove 'edges' parameter from as_adjacency_matrix()

### ISSUE THREAD

**Issue by szhorvat** [igraph/rigraph#815]
The `edges` parameter of `as_adjacency_matrix()` was promised to be removed in igraph 1.4.0, but this did not happen.

There is a warning:

```
warning("The `edges` argument of `as_adjacency_matrix` is deprecated; it will be removed in igraph 1.4.0")
```

 - [ ] Docs should mention the deprecation as well
 - [x] Update version in warning, as we're past 1.4.0
 - [ ] Remove parameter

This parameter is still functional as of 1.4.3, but we won't be able to provide it with C/igraph 0.10. I suggest removal in 1.5.0.

**Comment by szhorvat** [comment]
C/igraph 0.10 does not support the `edges` parameter. If you want to keep it, it needs to be reimplemented in R fully.

**Comment by maelle** [comment]
Ok, makes sense! I'll fix up #1080 after #1085 is finished.

**Comment by krlmlr** [comment]
Done now.
